src/locales/pages/en/about.mdx

```diff
@@ -80,13 +80,12 @@ The publication director is Mr. Boris RAVIGNON, Acting Chairman and CEO of ADEME
 ### Platform hosting
 
 This platform is hosted by :
-Netlify
-Inc 2325 3rd Street, Suite 296
-San Francisco, California 94107
+Vercel Inc.
+440 N Barranca Ave #4133
+Covina, CA 91723
 United States
 
-The site is hosted on Netlify. It's a static site served via a CDN: despite being domiciled in California, the site will be served to you [from
-europe](https://answers.netlify.com/t/is-there-a-list-of-where-netlifys-cdn-pops-are-located/855/2).
+The site is hosted on Vercel. It's a static site served via a CDN: despite being domiciled in California.
```

Comment:
despite being domiciled in California what ?

Reply:
I speake goode ingliche sank you